gitlab.py

```diff
@@ -44,6 +44,9 @@ def format_datetime(dt):
             repo["derniere_mise_a_jour"] = format_datetime(
                 repository["last_activity_at"]
             )
+            repo["derniere_modification"] = format_datetime(
```

Comment:
Maybe just add a comment in here saying that `derniere_modification` really ought to be the equivalent of GitHub `pushed_at` but GitLab does not expose such information for now.

Reply:
Thanks, added